index.html

```diff
@@ -2310,7 +2272,7 @@ <h4>Presentations Including Holder Claims</h4>
 mechanism as the <a>verifiable presentation</a>.
           </p>
 
-          <pre class="example nohighlight" title="A verifiable presentation, secured with an embedded proof, with a self-asserted verifiable credential">
```

Comment:
updated title to be consistent with previous example title changes

Reply:
-1 for removing securing examples from the specification. Please convert into an example that shows how it's secured.